### PR TITLE
Image/add

### PR DESCRIPTION
**Pull request by hewonjeong**
- Image post api

### CODE REVIEW

**Comment by festinalente91** [comment]
lgtm

**Comment by ovekyc** [comment]
Please separate lint fix and new code next time!

File: src/server/images/controllers.js [hunk 1]
```diff
@@ -88,5 +89,52 @@ export default {
...
+    .then((url) => {
+      return new Promise((resolve, reject) => {
+        const image = {
+          key: key,
+          url: url,
+          userKey: req.body.userKey,
+          caption: req.body.caption,
+          createdDate: currentTime.toISOString()
+        };
+        const idxImage = {key: key};
```
Comment (ovekyc) [inline src/server/images/controllers.js]: You can just write {key} instead of {key: key}

File: src/test/node/images/controllers.js [hunk 2]
```diff
@@ -96,3 +97,68 @@ test('get an image', t => {
...
 });
+test('Post image', t => {
+  const expected = {
+    status: 200,
+    message: 'success',
+    isImageInS3: true
+  };
+  const req = httpMocks.createRequest({
+    method: 'POST',
+    url: '/iamges',
```
Comment (ovekyc) [inline src/test/node/images/controllers.js]: /images

**Comment by ovekyc** [comment]
LGTM

**Comment by festinalente91** [comment]
lgtm

**Pull request merged**